test/test_torch.py

```diff
@@ -6082,8 +6082,13 @@ def compare_strides(s1, s2, div):
                 _test_helper(x, op, unary=True)
 
     @skipMeta
-    def test_dlpack_conversion(self, device):
-        x = torch.randn(1, 2, 3, 4, device=device, dtype=torch.float)
+    @dtypes(*torch.testing.get_all_dtypes())
+    def test_dlpack_conversion(self, device, dtype):
+        # DLpack does not explicitly support bool
+        # It does it through uint8 type
+        if dtype is torch.bool:
+            return
+        x = torch.randn(1, 2, 3, 4, device=device, dtype=torch.float).type(dtype)
```

Comment:
This is a little easier with `make_tensor()`:

```
t = make_tensor((1, 2, 3, 4,), device=device, dtype=dtype)
```

Reply:
thanks! will fix

Reply:
This week is still needed